src/App/Command/SlackNotifierCommand.php

```diff
@@ -149,44 +138,28 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         // Local Variable
-        $slackMessageCore = $slackMessageQA = $slackMessageCoreMembers = [];
+        $slackMessageQA = $slackMessageCoreMembers = [];
 
         $this->github = new Github($input->getOption('ghtoken'));
         $this->moduleChecker = new ModuleChecker($this->github);
         $this->moduleFetcher = new ModuleFetcher($this->github);
         $this->nightlyBoard = new NightlyBoard();
         $this->slack = new Slack($input->getOption('slacktoken'));
-        $this->slackChannelCore = $input->getOption('slackchannelCore');
         $this->slackChannelQA = $input->getOption('slackchannelQA');
 
         $title = ':preston::date: Welcome to the PrestHubot Report of the day :date:';
-        $slackMessageCore[] = $title;
         $slackMessageQA[] = $title;
 
         // Check Status
         $statusNightly = $this->checkStatusNightly();
-        $slackMessageCore[] = $statusNightly;
         $slackMessageQA[] = $statusNightly;
 
-        // Check if PR are need to merge
-        $slackMessageCore[] = $this->checkPRReadyToMerge();
-
-        // Check PR to review
-        $slackMessageCore[] = $this->checkPRReadyToReview();
-
-        // Need module releases
-        $slackMessageCore[] = $this->checkModuleReadyToRelease();
-
-        // Need module improvements
-        $slackMessageCore[] = $this->checkModuleImprovements();
-
```

Comment:
Please remove all unuseful methods/

Reply:
Are you sure? I removed only the notification sending and the message, but the other methods contain valuable logic that could be used later.

Reply:
If it's not used, it's dead. The logic and its code can be found thanks to Git & Github.

Reply:
Done in https://github.com/PrestaShop/presthubot/pull/52/commits/1d626007a61718b39af65664468544e9e4b47039